web/cobrands/northamptonshire/assets.js

```diff
@@ -516,4 +516,25 @@ fixmystreet.assets.add($.extend(true, {}, fixmystreet.assets.alloy_defaults, {
     }
 }));
 
+function ncc_check_streetlights() {
+    var relevant_body = OpenLayers.Util.indexOf(fixmystreet.bodies, 'Northamptonshire County Council') > -1;
```

Comment:
If you put `body: "Northamptonshire County Council"`  in `northants_defaults`, then you could use `northants_defaults.body` here.